configuration/concepts/total_joints.csv

```diff
@@ -0,0 +1,26 @@
+Uuid,Void/Retire,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Answers,Members
+f93d0420-4771-4988-bf61-8b9461ec4005,,Shoulder,Shoulder,Hombro,Hombro,,Misc,N/A,,
+245d28b0-d1c5-4b85-a2fd-94cdd610130e,,Hip,Hip,Cadera,Cadera,,Misc,N/A,,
+66e3e1d1-3daf-4ac3-8d79-85d0755abf00,,Knee,Knee,Rodilla,Rodilla,,Misc,N/A,,
+2ed9c8c0-1534-4d2d-a0d8-0987a9607d34,,Joint,Joint,Articulación,Articulación,,Misc,Coded,Shoulder; Hip; Knee,
+a11a2309-3600-4ea8-957c-18591654ad37,,Both,Both,Ambos,Ambos,,Misc,N/A,,
+11cd4cc3-0d09-4771-aac3-40e0ab4342b5,,Body Side,Body Side,Lado del cuerpo,Lado del cuerpo,,Misc,Coded,Right; Left; Both,
+55ad06d0-f5d7-4bfc-9a3e-8a017047da26,,Patient's Preference,Patient's Preference,Preferencia del paciente,Preferencia del paciente,,Misc,Coded,Right; Left,
+31420631-a70e-4658-9aeb-fdac4a4f83dc,,Duration of Pain,Duration of Pain,Duración del dolor,Duración del dolor,,ConvSet,N/A,,Hour(s); Weeks; Minutes; Months
+935c40a8-b737-44c1-b3a4-13ed2776c09d,,Normal,Normal,Normal,Normal,,Misc,N/A,,
+c45e2a69-8ebf-49ca-9138-b45783a149b1,,Varus,Varus,varo,varo,,Misc,N/A,,
+9c96e7a6-02f5-4424-b97a-3a879e2e3293,,Valgus,Valgus,valgo,valgo,,Misc,N/A,,
+6771f30b-75c6-497c-9eb0-def233d13987,,Joint Formation,Joint Formation,Formación conjunta,Formación conjunta,,Misc,Coded,Normal; Varus; Valgus,
+c9c81d97-f606-4b47-8859-a63344d7f7bf,,Range of Motion,Range of Motion,Rango de movimiento,Rango de movimiento,,Misc,Text,,
+7627ac92-4270-4c5c-b59d-6e1455ae6baf,,Current State of Disability/ Pain,Current State of Disability/ Pain,Estado actual de discapacidad/dolor,Estado actual de discapacidad/dolor,,Misc,Text,,
+1d5fb0d1-6d89-4755-9e0e-5acb14597d42,,Previous Treatment,Previous Treatment,Tratamiento previo,Tratamiento previo,,Misc,Text,,
+9406d455-5a43-48c7-beca-f65a199bc852,,Most Recent X-ray date,Most Recent X-ray date,Fecha de la radiografía más reciente,Fecha de la radiografía más reciente,,Misc,Text,,
```

Comment:
The datatype should be 'Date' rather than free 'text'

Reply:
Done